bugbug/model.py

```diff
@@ -455,16 +718,32 @@ def train(self, importance_cutoff=0.15, limit=None):
 
         print("Test Set scores:")
         # Evaluate results on the test set.
-        y_pred = self.clf.predict(X_test)
+        if is_multilabel:
+            y_pred = self.clf.predict(X_test.toarray())
+        elif is_binary:
+            y_pred = self.clf.predict(X_test)
 
         if is_multilabel:
             assert isinstance(
                 y_pred[0], np.ndarray
             ), "The predictions should be multilabel"
 
         print(f"No confidence threshold - {len(y_test)} classified")
+
         if is_multilabel:
             confusion_matrix = metrics.multilabel_confusion_matrix(y_test, y_pred)
+
+            for num, array in enumerate(confusion_matrix):
+                df_cm = pd.DataFrame(
+                    array,
+                    index=[self.class_names[num], "Other"],
+                    columns=[self.class_names[num], "Other"],
+                )
+                sn.set(font_scale=1)
+                sn.heatmap(df_cm, annot=True, annot_kws={"size": 16})  # font size
+                plt.title(self.class_names[num] + " Confusion Matrix")
+                plt.show()
```

Comment:
Can you show me the output of this? Is it that different from the command line output from `print_labeled_confusion_matrix`?
If it isn't, we can remove this code and remove the seaborn dependency